TM1py/Services/ViewService.py

```diff
@@ -205,3 +208,57 @@ def delete(self, cube_name: str, view_name: str, private: bool = False, **kwargs
         url = format_url("/api/v1/Cubes('{}')/{}('{}')", cube_name, view_type, view_name)
         response = self._rest.DELETE(url, **kwargs)
         return response
+
+    def search_subset_in_view(self, dimension_name: str = None, subset_name: str = None, cube_name: str = None, 
+                              include_elements: bool = True, **kwargs) -> Tuple[List[View], List[View]]:
+        """ Get all public and private views that utilize specified dimension subset
+
+        :param dimension_name: string, valid dimension name with subset to query
+        :param subset_name: string, valid subset name to search for in views
+        :param cube_name: str, optionally specify cube to search, otherwise will search all cubes
+        :param include_elements: false to return view details without elements, faster
+        :return: 2 Lists of TM1py.View instances: private views, public views
+        """
+
+        element_filter = ";$top=0" if not include_elements else ""
+        cube_filter = "('{}')".format(cube_name) if cube_name else ""
+        
+        private_views, public_views = [], []
+        for view_type in ('PrivateViews', 'Views'):
+            url = format_url(
+                "/api/v1/Cubes{}?$expand={}("
```

Comment:
Is it worth being more specific on the properties e.g Name only to retrieve for the cube if we're only interested in the views? By default, this is going to return the rules for each cube amongst other properties

Reply:
good catch, I had $select=name in an earlier test and somehow dropped it.